.github/workflows/test.yml

```diff
@@ -96,3 +96,33 @@ jobs:
       - name: Run tests
         run: |
           pytest -n auto
+  build-free-threading:
```

Comment:
We may want to add a release job also. Since Python 3.13.0rc1 and NumPy 2.1.0 are released, we should be able to ship wheels in the next release.

Fine to do in a follow-up, if you prefer.